src/api/index.spec.tsx

```diff
@@ -145,6 +145,36 @@ describe(__filename, () => {
     });
   });
 
+  describe('getVersionFile', () => {
```

Comment:
I thought we decided not to use specialized API functions given that it would provide little value.

Reply:
I thought we just decided not to create additional files for them, and to put them all into `api/index.tsx` for now. The value I see is that we get some type safety around the call because of the params, and also we have centralization of API calls. I assume the other method would be to inline the call to `callApi` inside other files (whatever our equivalent to sagas will be) and that could result in them sprinkled around the code.

Reply:
fair enough